qubes-rpc/admin.repos.List

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
+
+import dnf
+
+base = dnf.Base()
+
+base.read_all_repos()
+
+for repo in base.repos.all():
+	l = [repo.id, repo.name, 'enabled' if repo.enabled else 'disabled']
+	print('\0'.join(l))
```

Comment:
I think it is weird to use `\0` for field separation, but `\n` for record separation. But I see why you've done it this way.

Reply:
Yeah, I thought about using `:` for field separation, like traditional Unix record fields. The problem with that is that the name of repositories might include `:` and if we add another field to hold the URL in the future, that will _definitely_ include `:`.

I didn't want to use the same character to separate records as I used to separate fields. Hence, `\n`. I'm open to other suggestions, it certainly takes some getting used to.